libs/defi/oeth/src/bridge/components/BridgeCard.tsx

```diff
@@ -0,0 +1,471 @@
+import { useEffect, useState } from 'react';
+
+import {
+  Accordion,
+  AccordionDetails,
+  Box,
+  Card,
+  CardContent,
+  CardHeader,
+  Collapse,
+  Stack,
+  Typography,
+} from '@mui/material';
+import { TokenInput } from '@origin/defi/shared';
+import { ArrowButton, InfoTooltip } from '@origin/shared/components';
+import { ChainButton } from '@origin/shared/components';
+import {
+  contracts,
+  getNativeToken,
+  getTokenId,
+} from '@origin/shared/contracts';
+import { ChainlinkCCIP } from '@origin/shared/icons';
+import {
+  ApprovalButton,
+  isNativeCurrency,
+  TxButton,
+  useTxButton,
```

Comment:
you'd need to use `useTxButton` from `@origin/defi/shared` to plug to the new notifications
Note that I added a `useApprovalButton` that works the same way to replace the `ApprovalButton` with a `TxButton`

Reply:
> Note that I added a useApprovalButton that works the same way to replace the ApprovalButton with a TxButton

Is that important to use now?

Reply:
yes, I changed the notifications, these are the right hooks for displaying them